website/community/2-resources.mdx

```diff
@@ -43,7 +43,7 @@ See the <a href={require('@docusaurus/useBaseUrl').default('showcase')}>showcase
 - [meilisearch-docsearch](https://github.com/tauri-apps/meilisearch-docsearch) - Docusaurus plugin for [Meilisearch](https://www.meilisearch.com)
 - [@orama/plugin-docusaurus](https://github.com/askorama/orama/tree/main/packages/plugin-docusaurus) - [Orama](https://askorama.ai/) plugin for Docusaurus v2
 - [@orama/plugin-docusaurus-v3](https://github.com/askorama/orama/tree/main/packages/plugin-docusaurus-v3) - [Orama](https://askorama.ai/) plugin for Docusaurus v3
-- [@getcanary/docusaurus-pagefind](https://getcanary.dev/docs/integrations/docusaurus.html) - Create [Pagefind](https://pagefind.app/) index and use [Canary](https://github.com/fastrepl/canary) as UI primitives.
+- [@getcanary/docusaurus-theme-search-pagefind](https://getcanary.dev/docs/integrations/docusaurus.html) - Create [Pagefind](https://pagefind.app/) index and use [Canary](https://github.com/fastrepl/canary) as UI primitives.
```

Comment:
Your package name on that doc page hasn't been updated to reflect the new name and it's a theme, is this expected?

![CleanShot 2024-08-01 at 12 19 46](https://github.com/user-attachments/assets/9d894b72-d884-4ad7-8c2a-d27253f16857)

Reply:
Thank you for the pointer! It's now updated.